src/ApiProblem.php

```diff
@@ -250,14 +250,24 @@ protected function getDetail()
     }
 
     /**
```

Comment:
We've used the term "raw" elsewhere in the framework, including zend-inputfilter; this would at least be consistent.